c-example/example.c

```diff
@@ -0,0 +1,50 @@
+#include <stdio.h>
+
+#include "spglib.h"
+
+int main(void) {
+    SpglibDataset *dataset;
+    int i, j, size;
+    const char *wl = "abcdefghijklmnopqrstuvwxyz";
```

Comment:
What is stored here, recommended allocated size unless we are using safe string commands

Reply:
Yes! Done at 4821e31.